Prose/ProseLib/Sources/Mocks/Message.swift

```diff
@@ -0,0 +1,29 @@
+//
+// This file is part of prose-app-macos.
+// Copyright (c) 2022 Prose Foundation
+//
+
+import Foundation
+import ProseCoreTCA
+
+public extension Message {
+  static func mock(
+    from: JID = "jane.doe@prose.org",
+    id: String = "message-id",
```

Comment:
Does it make sense to have a static default id? Won't it cause conflicts?
I usually do `UUID().uuidString`, keeping only a subset if needed (for length).